packages/devtools_app/lib/src/framework/app_error_handling.dart

```diff
@@ -86,3 +107,55 @@ void reportError(
     );
   }
 }
+
+SingleMapping? _cachedJsSourceMapping;
+SingleMapping? _cachedWasmSourceMapping;
+
+Future<SingleMapping?> _fetchSourceMapping() async {
+  final cachedSourceMapping = preferences.wasmEnabled.value
+      ? _cachedWasmSourceMapping
+      : _cachedJsSourceMapping;
+
+  return cachedSourceMapping ?? (await _initializeSourceMapping());
+}
+
+Future<SingleMapping?> _initializeSourceMapping() async {
+  try {
+    final wasmEnabled = preferences.wasmEnabled.value;
+    final sourceMapUri = Uri.parse(
+      'main.dart.${wasmEnabled ? 'wasm' : 'js'}.map',
```

Comment:
same here use kIsWasm for the boolean check in the ternary

Reply:
Done!